src/api/WebLink.js

```diff
@@ -37,14 +36,14 @@ class WebLink extends Item {
      * Gets a Box weblink
      *
      * @param {string} id - Weblink id
-     * @param {(newItem: BoxItem) => void} successCallback - Function to call with results
```

Comment:
updated the type since there were issues with flow. this aligns with how it's written for `getFile` and `getFolder` methods